### ISSUE TITLE
Styles are not correctly switched. Error CORS

### ISSUE THREAD

**Issue by kosst-p** [darkreader/darkreader#6294]
Hello! Thank you for your work and your library!
But I have some problem and I wanted to ask a question :
I have page... some things on page made with ExtJs. Some styles are connected remotely - `<link rel="stylesheet" href="<%=g_app.getContentManager().getExternalLibrariesRoot()%>extjs/4.2.1/resources/css/ext-all-offer.css"/>`.
 I used `DarkReader.setFetchMethod( window.fetch )`.
When I activate the theme not all styles are processed:
![error1](https://user-images.githubusercontent.com/45897728/126080318-3591c206-48c4-4e0a-817b-ccef4e6634e9.png)
Then I decided to add headers:
```
window.DarkReader.setFetchMethod( fetcher );
function fetcher( url, options ) {
    return window.fetch( url, updateOptions( url, options ) );
}
function updateOptions( url, options ) {
    if ( typeof options === 'undefined' ) {
        var options = {};
    }
    options.mode = 'no-cors',
    options.headers = {
        'access-control-allow-origin': '*',
    }
    return options;
}
```
And I stopped getting CORS mistake. But the result has not changed. Styles still do not switch to a dark theme.
For the test, I set the expansion for the browser and decided to check everything works well 
![error2](https://user-images.githubusercontent.com/45897728/126080518-a50a3e3b-3d62-4af9-ad1c-8f0903e260f3.png)
Is it possible to make styles displayed correctly?
What could be the reason?
Thanks in advance for the answer

**Comment by Gusted** [comment]
So first off the Acces-control-allow-origin isn't a header that will have affect as it's an [response header](https://developer.mozilla.org/en-US/docs/Glossary/Response_header). Also `no-cors` isn't a mode you would like to use, please read what the [no-corse](https://developer.mozilla.org/en-US/docs/Web/API/Request/mode) mode does if you enable it.

> Is it possible to make styles displayed correctly?

That's up-to you. There's nothing we can do, this is just old good browsers implementation security features. However I would try to explain what's going on here and what changes possible could be made.

> What could be the reason?

It is because of [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) which TL;DR safe-guards you cannot make an fetch call to some resource on another domain/webpage. If you would like to bypass such behavior, it is the best to have on your site a 'proxy' to these resources so they would be loaded from your domain and not encounter any CORS issues. 

Otherwise you would need to find a way to request these resource from a server that explicitly enables the `Access-Control-Allow-Origin` with the `*` value on the response.

Regards,
Gusted

**Comment by kosst-p** [comment]
Wow! Thanks for such a quick answer !!!
If I understand you correctly, styles are not fully used because of the cors?  Because styles are on another domain? 
And if solve this problem, then the color topic will be the same as when using the expansion for the browser? 

Thanks,
Konstantin.

**Comment by Gusted** [comment]
> If I understand you correctly, styles are not fully used because of the cors? Because styles are on another domain?

Correct-ish, the network request that have to be made towards the resource(in this case the .css) are on another domain. And because of that the CORS kicks in and currently will error the request because it hasn't the correct headers to allow such cross-origin request.

> And if solve this problem, then the color topic will be the same as when using the expansion for the browser?

Yes.
